src/services/cron.js

```diff
@@ -1,18 +1,47 @@
 const cron = require("node-cron");
+const dotenv = require("dotenv")
+const userDB = require('../models/user');
+const paymentDB = require('../models/payment');
+const subscriptionDB = require('../models/subscription');
+const { db } = require('../models/user');
+const makePayment = require("./iyzico");
+
+dotenv.config({
+    path: "config.env",
+  });
+
+const price = process.env.PRICE;
+const interval = parseInt(process.env.INTERVAL);// 1 day of Interval for every succesfull subscriptions
 
 const scheduleSubscriptionPayment = () => {
   // Every day at 00:00:00
-  cron.schedule("0 0 0 * * *", () => {
+  cron.schedule("0 * * * * *", async() => {//checking every minute for test purpose!
     console.log("Running a task every day, date:", new Date());
+    const sub = await subscriptionDB.find();
 
-    /**
-     * TODO: Get records from Subscription collection
-     * We should have a logic to understand payment is needed or not.
-     * Then call makePayment service
-     * Note: It is ok to change Subscription schema if needed.
-     *
-     * */
+    if(sub){
```

Comment:
no need if clause here since we will loop over the results, something like below:
`const subscriptions = await subscriptionDB.find();
subscriptions.forEach();`